### ISSUE TITLE
feature request packing = simple

### ISSUE THREAD

**Issue by micrat** [ecmwf/ecmwf-opendata#17]
Hello,
apparently it is not available the MARS keyword
packing = simple
as described in:

[https://confluence.ecmwf.int/display/FCST/Implementation+of+IFS+Cycle+48r1#ImplementationofIFSCycle48r1-DefaultpackingtypesfordatafromDisseminationandMARS](https://confluence.ecmwf.int/display/FCST/Implementation+of+IFS+Cycle+48r1#ImplementationofIFSCycle48r1-DefaultpackingtypesfordatafromDisseminationandMARS)

Could you please implement in view of the cycle release 48r1?

**Comment by mmuscato** [comment]
Agreed with @micrat. This cycle release is live now. Would be nice to be able to use packing = simple.
